contracts/VaultV3.vy

```diff
@@ -598,9 +610,14 @@ def _add_strategy(new_strategy: address):
    log StrategyAdded(new_strategy)
 
 @internal
-def _revoke_strategy(old_strategy: address):
+def _revoke_strategy(old_strategy: address, force: bool):
    assert self.strategies[old_strategy].activation != 0, "strategy not active"
-   assert self.strategies[old_strategy].current_debt == 0, "strategy has debt"
+   loss: uint256 = 0
+
+   if self.strategies[old_strategy].current_debt != 0:
+    assert force, "strategy has debt"
+    loss = self.strategies[old_strategy].current_debt
+   
```

Comment:
need to decrease self.total_debt by loss amount here right

Reply:
https://github.com/flashfish0x/yearn-vaults-v3/pull/1/commits/13f0152deac86c0041ddce48f97557f0709f58e6